bodhi-server/bodhi/server/models.py

```diff
@@ -1558,30 +1556,26 @@ def update_test_cases(self, db):
 
         start = datetime.utcnow()
         log.debug(f'Querying the wiki for test cases of {self.nvr}')
-
-        wiki = MediaWiki(config.get('wiki_url'))
-        cat_page = f'Category:Package {self.package.external_name} test cases'
+        try:
+            wiki = MediaWiki(config.get('wiki_url'),
+                             user_agent=config.get('wiki_user_agent'))
+        except Exception:
```

Comment:
Would it be useful here to also store in the `ExternalCallException` the error message that was raised?